CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## [unreleased]
 
 ### Protocol Changes
-**No Changes**
+* The contract runtime has been upgraded to use the new Wasmtime-based runtime;
+* The contract runtime now allows for bulk memory instructions in WASM code.
```

Comment:
nit; correct capitalization is actually *Wasm* (see e.g. here: https://webassembly.org/)
```suggestion
* The contract runtime now allows for bulk memory instructions in Wasm code.
```